tests/rbd/test_rbd_trash_purge_schedule.py

```diff
@@ -49,7 +49,7 @@ def verify_schedule_add(
     log.debug(f"Verifying purge on pool {pool}")
     while iterations_remaining > 0:
         log.debug(f"Sleeping for {schedule_interval} mins with buffer")
-        time.sleep(schedule_interval * 60 + 5)
+        time.sleep(schedule_interval * 60 + 60)
```

Comment:
Why did you increase the sleep time from 5 to 60?

Reply:
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-T4XM6I/Test_to_verify_trash_purge_schedule_0.log
added buffer is not enough trash list still holds some entries after trash purge also.
hence after increasing sleep buffer issue not found